examples/vnet/doca_encap_table.c

```diff
@@ -104,22 +109,25 @@ int doca_encap_table_add_id(struct doca_flow_encap_action *ea)
 {
     struct encap_table_key key = {0};
     int id = 0;
-    if ((id = doca_encap_table_get_id(ea)) >= 0)
+    if ((id = doca_encap_table_get_id(ea)) >= 0) {
+        encap_table_ins->entries[id].refcnt++;
```

Comment:
this should be done under lock

Reply:
Done.

Reply:
there is a lock inside doca_encap_table_get_id

Reply:
Done.